core/data/src/main/java/com/iteneum/ItemList.kt

```diff
@@ -0,0 +1,7 @@
+package com.iteneum
+
+data class ItemList(
```

Comment:
Please be more descriptive, I mean, which kind of item list is going to be this data class?

Reply:
this is not mine you will see on the PR description sorry  , please wait will update in few minutes